library/lib_window.ks

```diff
@@ -0,0 +1,26 @@
+
+
+@lazyglobal off.
+
+function create_window{
+	parameter x,y,w,h.
+	// w and h are INTERNAL width and height
```

Comment:
Why you choose the internal w and h? Isn't it easier for a program that uses your library to go: "I can give you this space, draw what ever you want here"?